black/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6-alpine3.7
+
+RUN addgroup -S black && adduser -D -S black black && \
+    apk add --no-cache bash && \
```

Comment:
Where is this used?

Reply:
What? The user? Is necessary to have a cache directory in it's home. And it's used right [here](https://github.com/kiwicom/dockerfiles/pull/20/files#diff-7c3ef15bbb1c74f6e6c8643822c0a1f5R11)

Reply:
bash

Reply:
Yep 🤦‍♂️ I was looking at the wrong line. And indeed is not necessary. I don't know why I did that... I tried the image without it and it works fine